workspace/build_manifest_test.go

```diff
@@ -0,0 +1,134 @@
+package workspace
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/yourbase/narwhal"
+	"github.com/yourbase/yb/runtime"
+)
+
+func TestExecPhase_EnvironmentVariablesDotEnv(t *testing.T) {
+	type fields struct {
+		Name         string
+		Dependencies ExecDependencies
+		Container    narwhal.ContainerDefinition
+		Commands     []string
+		Ports        []string
+		Environment  map[string][]string
+		HostOnly     bool
+	}
+	type args struct {
+		ctx     context.Context
```

Comment:
Rather than placing the Context in the table test data, pass it in as part of the test loop.

Reply:
OK!